samples/boards/nrf52/mesh/onoff_level_vnd_app/src/common.h

```diff
@@ -0,0 +1,67 @@
+/*  Bluetooth: Mesh Generic OnOff, Generic Level & Vendor Models
+ *
+ * Copyright (c) 2018 Vikrant
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _COMMON_H
+#define _COMMON_H
+
+#define  SETB(x, y)   (x |= (1<<y))     /* for o/p */
+#define  CLRB(x, y)   (x &= (~(1<<y)))  /* for o/p */
+#define  TGLB(x, y)   (x ^= (1<<y))     /* for o/p */
+#define  CHECKB(x, y) (x &  (1<<y))     /* for i/p */
+
+#include <misc/printk.h>
+#include <settings/settings.h>
+
+#include <misc/byteorder.h>
+#include <nrf.h>
+#include <device.h>
+#include <gpio.h>
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/l2cap.h>
+#include <bluetooth/hci.h>
+#include <bluetooth/mesh.h>
+#include <stdio.h>
+
+#include <board.h>
+
+#include "nrf52840.h"
+#include "board.h"
+
+#include "ble_mesh.h"
+#include "device_composition.h"
+#include "publisher.h"
```

Comment:
Please move all the includes to the c-files that need them. This kind of indirect inclusion is not really used in the tree. The only exception is public header files and even then the only includes they have is for things that's part of the header file itself (here neither one of these is true).